vue/components/ui/molecules/scrollable-items/scrollable-items.vue

```diff
@@ -0,0 +1,246 @@
+         * scroll wheel. Negative values inverts the default direction of the scroll.
+         */
+        scrollSpeed: {
+            type: Number,
+            default: 10
+        },
+        /**
+         * Props for the arrow button icon components.
+         */
+        buttonIconProps: {
+            type: Object,
+            default: () => ({})
+        }
+    },
+    data: function() {
+        return {
+            selectedData: this.selected,
+            scrollableWidth: 0
+        };
+    },
+    computed: {
+        itemsContainerStyle() {
+            return {
+                "text-align": this.alignment
+            };
+        },
+        _buttonIconProps() {
+            return {
+                borderRadius: 0,
+                iconStrokeWidth: 3,
+                ...this.buttonIconProps
+            };
+        },
+        showArrows() {
+            const hideArrows = this.autoHideArrows && this.scrollableWidth <= 0;
+            return this.arrows && !hideArrows;
+        },
+        selectedIndex() {
+            return this.items.findIndex(item => item.value === this.selectedData);
+        },
+        isPreviousDisabled() {
+            return this.selectedIndex <= 0;
+        },
+        isNextDisabled() {
+            return this.selectedIndex >= this.items.length - 1;
+        }
+    },
+    watch: {
+        items() {
+            this.$nextTick(() => this.calculateScrollableWidth());
+        },
+        selected(value) {
+            this.selectedData = value;
+        },
+        selectedData(value) {
+            this.$nextTick(() => this.snapSelectedToCenter());
+            this.$emit("update:selected", value);
+        }
+    },
+    created: function() {
+        window.addEventListener("resize", this.onResize);
+    },
+    destroyed: function() {
+        window.removeEventListener("resize", this.onResize);
+    },
+    methods: {
+        itemClasses(item, index) {
+            const base = {};
+            base[`item-${item.value}`] = true;
+            if (item.value === this.selectedData) base.selected = true;
+            return base;
+        },
+        calculateScrollableWidth() {
+            if (!this.$refs["items-container"]) return;
+            this.scrollableWidth = this.$refs["items-container"].scrollLeftMax;
+        },
+        snapSelectedToCenter() {
+            if (this.selectedIndex < 0) return;
+            this.$refs["items-container"].childNodes[this.selectedIndex].scrollIntoView({
+                behavior: "smooth",
+                inline: "center"
+            });
+        },
+        onResize(event) {
+            this.calculateScrollableWidth();
+        },
+        onPreviousClick(event) {
+            this.selectedData = this.items[this.selectedIndex - 1].value;
+        },
+        onItemsContainerWheel(event) {
+            this.$refs["items-container"].scrollLeft += event.deltaY * -this.scrollSpeed;

```

Comment:
Standard: `-1 * this.scrollSpeed`. Either that or `-= event.deltaY * this.scrollSpeed`